<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>11</y>
      <width>731</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="main_Layout">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
       <item>
        <layout class="QVBoxLayout" name="online_Layout">
         <item>
          <widget class="QLabel" name="Users_Lbl">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Online users</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listUsers">
           <item>
            <property name="text">
             <string>User1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>User2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>User3</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="tasks_Layout">
         <item>
          <widget class="QLabel" name="Tasks_lbl">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tasks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QueueTable" name="tableTasks"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="btn_Layout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="deleteBtn">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QueueTable</class>
   <extends>QTableWidget</extends>
   <header>QueueTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
